lib/capistrano/stretcher/tasks/rails.rake

```diff
@@ -1,5 +1,11 @@
+# Original source about asset_precompile: https://github.com/capistrano-plugins/capistrano-faster-assets/blob/master/lib/capistrano/tasks/faster_assets.rake
```

Comment:
Could you use `assets:precompile` or `assets precompile`? `asset_precopmile` is not avaiable default rails tasks/terms.